app/src/main/java/com/simplecity/amp_library/ui/screens/nowplaying/PlayerFragment.java

```diff
@@ -771,7 +771,7 @@ public void showRingtoneSetMessage() {
     }
 
     @Nullable
-    private Song getSong() {
+    public static Song getSong() {
```

Comment:
I wouldn't make a method `static` that is on a Fragment class - more than one fragment instance can exist at the same time and potentially introduce strange behavior later if this method is altered with the assumption that it only alters the current fragment instance.

But - it looks like the method is currently just a helper method to access the `static` service binder via the utils classes. So the contents of the current method are safe to call from a static method.

So, I would actually move this entire `getSong()` method to the `MusicServiceConnectionUtils` class and make it static there instead. So instead of calling `PlayerFragment.getSong()` you'd call `MusicServiceConnectionUtils.getSong()`.

Reply:
I sorted of implemented something similar to what you mentioned here, on the other PR (#91), 

https://github.com/CUTR-at-USF/MUSER/blob/ecd407357b412dfb5fcd92e7f1064b9a0e94e811/app/src/main/java/edu/usf/sas/pal/muser/util/SongUtils.java 

I could have another PR renaming this file and call it here once they are merged if that's okay with you.

Reply:
Sounds good - let's address this in https://github.com/CUTR-at-USF/MUSER/pull/91. I added another TODO item in the PR description there.